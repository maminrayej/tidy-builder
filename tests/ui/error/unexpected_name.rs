#[derive(tidy_builder::Builder)]
struct MyStruct {
    #[builder(eac = "arg")]
    args: Vec<String>,
}

fn main() {}
