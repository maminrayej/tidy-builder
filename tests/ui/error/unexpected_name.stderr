error: Unknown attribute
 --> tests/ui/error/unexpected_name.rs:3:15
  |
3 |     #[builder(eac = "arg")]
  |               ^^^^^^^^^^^
